cci_nepal/pipeline/classification_model/model_save_run/model_run.py

```diff
@@ -0,0 +1,169 @@
+from sklearn.compose import ColumnTransformer
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.preprocessing import RobustScaler
+from sklearn.multioutput import MultiOutputClassifier
+from sklearn.linear_model import LogisticRegression
+import pickle
+from pathlib import Path
+
+# Project libraries
+import cci_nepal
+from cci_nepal.getters.classification_model import get_data as grd
+from cci_nepal.pipeline.classification_model import data_manipulation as dm
+from cci_nepal.pipeline.classification_model import model_tuning_report as mtr
+from cci_nepal import config
+
+# %%
+# Set the project directory
+project_dir = cci_nepal.PROJECT_DIR
+
+# %%
+# Get parameters from config file
+b_features = config["final_model"]["basic_model_features"]
+nb_features = config["final_model"]["non_basic_model_features"]
+
+# %%
+# Read data and feature names
+test_hill = grd.read_test_hill_data()
+test_terai = grd.read_test_terai_data()
+column_names = grd.get_lists(f"{project_dir}/cci_nepal/config/column_names.csv")
+select_features = grd.get_lists(f"{project_dir}/cci_nepal/config/select_features.csv")
+
+# %%
+# Combine test sets and shuffle
+test = pd.concat([test_hill, test_terai], ignore_index=True)
+test = shuffle(test, random_state=1)
+
+# %%
+# Lowercase values
+test = test.applymap(lambda s: s.lower() if type(s) == str else s)
+
+# %%
+# Data transformations and feature creation
+dm.transform_sets(test, column_names)
+dm.feature_creation(test)
+
+# %%
+# X split
+X_test = test[select_features]
+
+# %%
+# Define the transformations to be made
+transformer = mtr.col_transformer(None)
+
+# %%
+# Apply column transformer
+X_test_transform = transformer.fit_transform(X_test)
+
+# %%
+# Assign back to dataframes - to have feature names back
+X_test_transform = pd.DataFrame(
+    X_test_transform, columns=list(transformer.get_feature_names_out())
+)
+
+
+# %%
+# Reduce to just chosen features for basic and non-basic
+X_test_basic = X_test_transform[b_features].copy()
+X_test_non_basic = X_test_transform[nb_features].copy()
+
+
+# %%
+# Add folder if not already created
+Path(f"{project_dir}/outputs/models/").mkdir(parents=True, exist_ok=True)
+
+# %%
+# Loading models (best performing)
+basic_model = pickle.load(
+    open(f"{project_dir}/outputs/models/final_classification_model_basic.sav", "rb")
+)
+
+non_basic_model = pickle.load(
+    open(f"{project_dir}/outputs/models/final_classification_model_non_basic.sav", "rb")
+)
+
+# %%
+# Predict on test set
+y_pred_basic = basic_model.predict_proba(X_test_basic)
+y_pred_non_basic = non_basic_model.predict_proba(X_test_non_basic)
+
+# %%
+# Items, basic and non-basic divide
+nfri_items = column_names[37:]
+basic = nfri_items[0:11]
+non_basic = nfri_items[11:]
+
+# %%
+# Create dataframes with model inputs and predictions for basic and non basic
+basic_preds = mtr.create_predictions_files(
+    y_pred_basic,
+    basic,
+    X_test,
+    [
+        "House_Material",
```

Comment:
For this file lets keep the columns consistent so either all lowercase or all uppercase first letters.